chemprop/cli/utils/parsing.py

```diff
@@ -115,7 +115,7 @@ def make_datapoints(
     V_fss: list[list[np.ndarray] | list[None]] | None,
     E_fss: list[list[np.ndarray] | list[None]] | None,
     V_dss: list[list[np.ndarray] | list[None]] | None,
-    features_generators: list[VectorFeaturizer[Mol]] | None,
+    features_generators: list[str] | None,
```

Comment:
what if we just rename this to `molecule_featurizers`? Seems like it would be a more descriptive name to clients of this function, even if we retain the `features_generators` variable name internally.

Reply:
Agreed. As I was working on #876, I felt this change would make the code more clear. If that PR doesn't end up going through, I can port those name changes to here.

Reply:
This change has been confirmed and will be made in #876.